forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/SendSubmissionToODSDelegate.java

```diff
@@ -87,11 +90,15 @@ private void sendSubmissionToODS(DelegateExecution execution) throws IOException
         ObjectMapper objectMapper = new ObjectMapper();
         String json = "";
         if (objectKeycase.equals("snake_case")) {
-            json = objectMapper.writeValueAsString(convertToSnake(values));
-        } else {
-            json = objectMapper.writeValueAsString(values);
+            values = convertToSnake(values);
+        }
+        if (replaceText != null) {
+            String[][] replaceArray = parseStringToArray(replaceText.split("\\s*,\\s*"));
+            if (replaceArray.length > 0) {
+                values = replaceTextAll(values, replaceArray);
+            }
         }
-        System.out.println(json);
```

Comment:
This line proved to be very useful for debugging purposes. I'd leave it here.

Reply:
very next line is that prints json value anyway if you pass debug true. That's why remove additional log here.
```
if(debug) {
    System.out.println("Sending values to ODS: " + json);
}
```

Reply:
Oh, you're right! I thought that on is the one. No worries then